src/Monthpicker.js

```diff
@@ -261,10 +229,6 @@ class MonthPicker extends React.Component {
       : { month: getMonth(date) + 1, year: getYear(date) }
```

Comment:
Is this `dateFormat` prop now necessary? Because the component is fully controlled and depends on `year` and `month`, would anybody want to get a formatted date out and then parse it to pass the correct props into the monthpicker again?

Not necessary to change this, just a thought.